app/modules/settings/components/displayed_links.svelte

```diff
@@ -1,46 +1,57 @@
 <script>
   import { I18n } from '#user/lib/i18n'
-  import { linksClaimsPropertiesByCategory } from '#entities/lib/entity_links'
+  import { getPropertiesFromWebsitesNames, getWebsitesNamesFromProperties, websitesByCategoryAndName } from '#entities/lib/entity_links'
   import { onChange } from '#lib/svelte/svelte'
   import Flash from '#lib/components/flash.svelte'
   import { debounce } from 'underscore'
 
-  const { bibliographicDatabases, socialNetworks } = linksClaimsPropertiesByCategory
+  const { bibliographicDatabases, socialNetworks } = websitesByCategoryAndName
 
-  let linksSettings = app.user.get('customProperties') || []
+  let customProperties = app.user.get('customProperties') || []
+  let stringifiedSavedCustomProperties = JSON.stringify(customProperties)
+
+  let selectedWebsites = getWebsitesNamesFromProperties(customProperties)
   let flash
 
   async function updateCustomProperties () {
     flash = null
     try {
-      await app.request('user:update', {
-        attribute: 'customProperties',
-        value: linksSettings
-      })
+      const stringifiedProperties = JSON.stringify(customProperties)
+      if (stringifiedProperties !== stringifiedSavedCustomProperties) {
+        stringifiedSavedCustomProperties = stringifiedProperties
+        await app.request('user:update', {
+          attribute: 'customProperties',
+          value: customProperties
+        })
+      }
     } catch (err) {
       flash = err
     }
   }
 
   const lazyUpdate = debounce(updateCustomProperties, 1000)
-  $: onChange(linksSettings, lazyUpdate)
+  function onSelectionChange () {
+    customProperties = getPropertiesFromWebsitesNames(selectedWebsites)
+    lazyUpdate()
+  }
+  $: onChange(selectedWebsites, onSelectionChange)
 </script>
 
 <fieldset>
   <legend>{I18n('bibliographic databases')}</legend>
```

Comment:
Now that checkboxes links becomes numerous, wouldnt make it more readable to upgrade 

```
  <h3>{i18n('Displayed links')}</h3>
```

to `h2` element ?

```
  <h2>{i18n('Displayed links')}</h2>
``` 

and subtitles from

```
  <legend>{I18n('bibliographic databases')}</legend>
```

to full fledged title element:

```
  <h3>{I18n('bibliographic databases')}</h3>
```

Reply:
I might be wrong, but I think the a11y is better with the `<legend>` labeling the fieldset. I made those headers more prominent in 01dd77bd5, but without going for `h2`, as that's already the header level of the page (`<h2 class="first-title">{I18n('display_name')}</h2>`)